src/config.rs

```diff
@@ -221,6 +229,10 @@ impl Config {
             self.build_bin = Inferrable::Inferred(None);
             self.build_lib = Inferrable::Inferred(false);
             self.cfg_test = false;
+            self.build_command = None;
+        } else if !self.build_on_save {
```

Comment:
Instead of requiring build_on_save, you could set build_on_save to true if there is a build command

Reply:
Will do